pkg/web/routes.go

```diff
@@ -104,6 +109,8 @@ func (wa *WebAPI) AddNotifier(n notifiers.Notifier) {
 // Router defines the routes for the application.
 func (wa *WebAPI) Router() *mux.Router {
 	r := mux.NewRouter()
+	r.HandleFunc("/logevents", wa.logevents).Methods("GET")
```

Comment:
Maybe it's just the diff, but are these routes unversioned? What if we added a version segment either before/after that would allow us to safely make changes to the endpoints?

Reply:
I added v1 for the first release